test/end-to-end/test_pkg_bulkupload.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#
+# Basic set of tests for the hab pkg bulkupload command
+#
+# Assumptions:
+# 1. ACCEPTANCE_HAB_AUTH_TOKEN or HAB_AUTH_TOKEN Environment variables are set and valid
+# 2. ${CACHE_DIR} can be set to a writable location on the filesystem
+# 3. non zero exit code from each command implies success
+
+# TODO: Future improvement would be to create a clean room Builder or origin.
+# Doing so would allow for more accurate validation of the bulkupload by interrogating
+# the Builder. Of course this introduces significant setup time cost..
+
+set -euo pipefail
+
+export HAB_NOCOLORING=true
+export HAB_NONINTERACTIVE=true
+export HAB_ORIGIN="testbulkupload"
```

Comment:
It would be great if we could use a more general test origin instead of specific to bulkupload.. I believe there has been some discussion already of a "habitat-test" origin